backend/main/models/vote.go

```diff
@@ -69,30 +70,31 @@ const (
 
 func GetVotesForAddress(
 	db *s.Database,
-	start, count int,
+	order shared.OrderedPageParams,
```

Comment:
Could we use consistent naming for this parameter? It looks like it is either being called `order` or `params`? Let's pick one and stick to it. I am thinking `params` since the struct contains more than just the order, or if we really wanted to be more clear maybe `pageParams`

Reply:
will use `pageParams`